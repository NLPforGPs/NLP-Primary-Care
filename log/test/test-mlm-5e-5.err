cpu-bind=MASK - bp1-gpu013, task 4294967295 4294967295 [0]: mask 0x4040 set

CondaValueError: prefix already exists: /user/home/zu20361/.conda/envs/NLP_GP

[nltk_data] Downloading package wordnet to
[nltk_data]     /user/home/zu20361/nltk_data...
[nltk_data]   Package wordnet is already up-to-date!
[nltk_data] Downloading package punkt to
[nltk_data]     /user/home/zu20361/nltk_data...
[nltk_data]   Package punkt is already up-to-date!
Some weights of the model checkpoint at microsoft/BiomedNLP-PubMedBERT-base-uncased-abstract were not used when initializing BertForMaskedLM: ['cls.seq_relationship.bias', 'cls.seq_relationship.weight']
- This IS expected if you are initializing BertForMaskedLM from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing BertForMaskedLM from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
2021-12-08 12:51:42,671 - /user/home/zu20361/NLP-Primary-Care/./run_plms.py[line:150] - INFO: Predicting...
2021-12-08 12:51:42,743 - /user/home/zu20361/.conda/envs/NLP_GP/lib/python3.9/site-packages/datasets/builder.py[line:377] - WARNING: Using custom data configuration default-a2d0abb6534a5099
0 examples [00:00, ? examples/s]2021-12-08 12:51:42,950 - /user/home/zu20361/.cache/huggingface/modules/datasets_modules/datasets/transcript_evaldataset/4c048f6bc1501180d0c57dc4d2b8c0d9958946e6a0943b3c5b80a90f1d04b330/transcript_evaldataset.py[line:59] - INFO: Generating examples...
                                  0%|          | 0/1 [00:00<?, ?it/s]100%|██████████| 1/1 [00:00<00:00, 289.90it/s]
2021-12-08 12:51:43,056 - /user/home/zu20361/NLP-Primary-Care/./run_plms.py[line:169] - INFO: MLM method.....
  0%|          | 0/1 [00:00<?, ?ba/s]100%|██████████| 1/1 [00:02<00:00,  2.08s/ba]100%|██████████| 1/1 [00:02<00:00,  2.09s/ba]
/user/home/zu20361/NLP-Primary-Care/nn_model/desc_classifier.py:132: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
  logits = logits[index]

real	1m24.000s
user	0m45.139s
sys	0m33.650s
