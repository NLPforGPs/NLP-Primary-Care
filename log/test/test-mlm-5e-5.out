JOB ID: 243199
Working Directory: /user/home/zu20361/NLP-Primary-Care
Warning: you have pip-installed dependencies in your environment file, but you do not list pip itself as one of your conda dependencies.  Conda may not use the correct pip to install your packages, and they may end up in the wrong place.  Please add an explicit pip dependency.  I'm adding one for you, but still nagging you.
Start Time: Wed Dec 8 12:51:28 GMT 2021
loading codes from PE...
Downloading and preparing dataset description_dataset/default (download: Unknown size, generated: 2.07 MiB, post-processed: Unknown size, total: 2.07 MiB) to /user/home/zu20361/.cache/huggingface/datasets/description_dataset/default-a2d0abb6534a5099/0.0.0/4c048f6bc1501180d0c57dc4d2b8c0d9958946e6a0943b3c5b80a90f1d04b330...
Dataset description_dataset downloaded and prepared to /user/home/zu20361/.cache/huggingface/datasets/description_dataset/default-a2d0abb6534a5099/0.0.0/4c048f6bc1501180d0c57dc4d2b8c0d9958946e6a0943b3c5b80a90f1d04b330. Subsequent calls will reuse this data.
classification_report:
                 precision    recall  f1-score   support

          blood       0.29      0.62      0.40         8
 cardiovascular       0.40      0.78      0.53        32
      digestive       0.65      0.59      0.62        44
            ear       0.77      0.91      0.83        11
      endocrine       0.34      0.83      0.49        24
            eye       0.33      0.60      0.43         5
         female       0.47      0.50      0.48        14
        general       0.05      0.57      0.09        14
           male       0.62      0.71      0.67         7
musculoskeletal       0.85      0.77      0.81        65
   neurological       0.23      0.55      0.33        20
      pregnancy       0.22      0.18      0.20        11
  psychological       0.63      0.80      0.71        50
    respiratory       0.54      0.81      0.65        37
           skin       0.77      0.75      0.76        32
     urological       0.62      0.72      0.67        18

      micro avg       0.41      0.71      0.52       392
      macro avg       0.49      0.67      0.54       392
   weighted avg       0.57      0.71      0.61       392
    samples avg       0.44      0.74      0.52       392

f1_score 0.5408475690147503
roc_auc 0.8708247444457625
End Time: Wed Dec 8 12:52:52 GMT 2021
